packages/iris-grid/src/formatters/TableColumnFormatter.ts

```diff
@@ -0,0 +1,69 @@
+/* eslint class-methods-use-this: "off" */
+/* eslint no-unused-vars: "off" */
```

Comment:
Curious why this rule is ignore. Can't think of any reason unused vars would be needed in a file

Reply:
I think it might've been for params when it was a js file - for some reason it complains about unused params in JS, in TS it does not. I've removed since this file is now ts.